p2panda-rs/src/document/document_view_id.rs

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use std::str::FromStr;
+
+use serde::{Deserialize, Serialize};
+
+use crate::hash::{Hash, HashError};
+use crate::Validate;
+
+/// The identifier of a document view.
+///
+/// Contains the hashes of the document graph tips which is all the information we need to reliably
```

Comment:
As we are not (really) dealing with time I'd maybe say "to reconstruct a specific version of a document"